### PR TITLE
New EAR artifacts providing Metrics with embedded Alerting

### PR DESCRIPTION
**Pull request by jshaughn**
- Remove JMS listening between metrics and alerting, they now communicate directly.
- Update name of metrics distribution
  - artifact hawkular-metrics-component-dist-VERSION.ear deployed as hawkular-metrics.ear
- Move publishing cache under hawkular-alerts container
- Up to final versions of deps
- Add new alerting cache

### CODE REVIEW

**Comment by jshaughn** [comment]
@pilhuhn Please review and merge and do release magic.

File: hawkular-rest/hawkular-listener/pom.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
     <dependency>
       <groupId>org.hawkular.alerts</groupId>
       <artifactId>hawkular-alerts-api</artifactId>
-      <scope>provided</scope>
+      <!-- <scope>provided</scope>  -->
```
Comment (pilhuhn) [inline hawkular-rest/hawkular-listener/pom.xml]: While I like doing that myself, the commented out stuff should not be committed.

**Comment by jshaughn** [comment]
@pilhuhn DoNotMerge, I found a problem.

**Comment by jshaughn** [comment]
@pilhuhn I fixed the issue, so merge is OK by me.

**Pull request merged**